pkg/controllers/execution/execution_controller.go

```diff
@@ -292,11 +292,11 @@ func (c *Controller) updateAppliedCondition(work *workv1alpha1.Work, status meta
 	})
 }
 
-func (c *Controller) eventf(object *unstructured.Unstructured, eventtype, reason, messageFmt string, args ...interface{}) {
+func (c *Controller) eventf(object *unstructured.Unstructured, eventType, reason, messageFmt string, args ...interface{}) {
 	ref, err := helper.GenEventRef(object)
 	if err != nil {
-		klog.Errorf("ignore event(%s) as failed to build event reference for: kind=%s, %s/%s due to %v", reason, object.GetKind(), object.GetNamespace(), object.GetName(), err)
+		klog.Errorf("ignore event(%s) as failed to build event reference for: kind=%s, %s due to %v", reason, object.GetKind(), klog.KObj(object), err)
```

Comment:
Should we do the same way in other places?

Reply:
I have the same doubts and often see this message in the log: `(/xxx) `. Perhaps we can create a work issue that records the modification of similar problems in all code modules.

Reply:
Good question.
I have an idea in my mind to implement one function for generating the `key` just for logging.
I'll try to do this and make a benchmark with `Klog`.